src/controllers/document_upload/document.upload.controller.ts

```diff
@@ -110,7 +110,7 @@ const getFileSizeLimitExceededCallback = (req: Request,
   return async (filename: string, maxInBytes: number) => {
     const maxInMB: number = getMaxFileSizeInMB(maxInBytes);
     logger.debug("File limit " + maxInMB + "MB reached for file " + filename);
-    const errorMsg: string = `${UploadErrorMessages.FILE_TOO_LARGE} ${maxInMB} MB`;
+    const errorMsg = `${UploadErrorMessages.FILE_TOO_LARGE} ${maxInMB} MB`;
```

Comment:
Removing the type information here seems to be a backwards step. Is there a rule that says this is a bad thing to do?

Reply:
Not really in previous PRs - it has been recommended that we removed types in this situation.

Reply:
It is ok in this situation, if you are assigning a string literal to a const/let/var as it's obvious what it is. if we were assigning the result of a function then we would need to specify :string to make it clear. There's some stuff on google about it, best practices perhaps.

Reply:
The rule for this seems to be true or false with no additional paramters to refine it, so I'll have to just switch it off.

Reply:
rule looks ok to me, we don't need to declare type when assigning a string literal.

Reply:
Yep, agreed we should leave the rule.